<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusteriBilgileriFormu</class>
 <widget class="QDialog" name="MusteriBilgileriFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Müşteri Bilgileri</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../materials.qrc">
    <normaloff>:/icons/Icons/ssl_tls_manager.png</normaloff>:/icons/Icons/ssl_tls_manager.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>278</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_TTemizle">
      <property name="text">
       <string>Tümünü Temizle</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/trash-empty.png</normaloff>:/icons/Icons/trash-empty.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Kaydet">
      <property name="text">
       <string>Kaydet</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/Ok.png</normaloff>:/icons/Icons/Ok.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_Cikis">
      <property name="text">
       <string>Çıkış</string>
      </property>
      <property name="icon">
       <iconset resource="../materials.qrc">
        <normaloff>:/icons/Icons/No.png</normaloff>:/icons/Icons/No.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_Adi">
      <property name="text">
       <string>Adı:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_Adi"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_Kodu">
      <property name="text">
       <string>Soyadı:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_Kodu"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>TC Kimlik:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Telefon:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Tahsilat:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>107</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../materials.qrc">:/icons/Icons/ssl_tls_manager.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Müşteri Ekle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../materials.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pb_Cikis</sender>
   <signal>clicked()</signal>
   <receiver>MusteriBilgileriFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Kaydet</sender>
   <signal>clicked()</signal>
   <receiver>MusteriBilgileriFormu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_Adi</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MusteriBilgileriFormu</receiver>
   <slot>degisiklikyapildi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_Kodu</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MusteriBilgileriFormu</receiver>
   <slot>degisiklikyapildi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MusteriBilgileriFormu</receiver>
   <slot>degisiklikyapildi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MusteriBilgileriFormu</receiver>
   <slot>degisiklikyapildi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MusteriBilgileriFormu</receiver>
   <slot>degisiklikyapildi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>degisiklikyapildi()</slot>
 </slots>
</ui>
